### PR TITLE
Twig Extension, remove type array, add .twig ext, add none filter

### PR DESCRIPTION
**Pull request by vonboth**

### CODE REVIEW

**Comment by WyriHaximus** [comment]
I'm not inclined to reject this PR but interested in why the removal of the array type hint.

**Comment by vonboth** [comment]
Hi
if you follow the link you will see the error PHP shows because of the type hint.
The function expects an array but the Token parser passes in a string.
Removing the type hint and solves this. But I haven't checked any side effects...

File: src/Lib/Twig/Extension/Strings.php [hunk 1]
```diff
@@ -40,6 +40,7 @@ public function getFilters()
             new \Twig_SimpleFilter('isMultibyte', 'Cake\Utility\String::isMultibyte'),
             new \Twig_SimpleFilter('utf8', 'Cake\Utility\String::utf8'),
             new \Twig_SimpleFilter('ascii', 'Cake\Utility\String::ascii'),
+            new \Twig_SimpleFilter('none', function($string) {return;}),
```
Comment (WyriHaximus) [inline src/Lib/Twig/Extension/Strings.php]: Could you fix the code style issue here? https://travis-ci.org/WyriHaximus/TwigView/jobs/144661875#L224-L232

**Comment by WyriHaximus** [comment]
> The function expects an array but the Token parser passes in a string.
> Removing the type hint solves this. But I haven't checked any side effects...

Right that makes sense, I'll merge this PR and tag a new release after you fixed the code style issue. If you don't prefer to do that I gladly merge it and do it no self :smile: .

**Comment by vonboth** [comment]
Hej, all done. Cheers

**Comment by WyriHaximus** [comment]
Thanks! I'll tag it later today :+1:

**Comment by WyriHaximus** [comment]
Tagged: https://github.com/WyriHaximus/TwigView/releases/tag/3.2.2

**Pull request merged**